{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "children": [],
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:ibm:cognos_controller:10.4.1:*:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:ibm:cognos_controller:10.4.2:*:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:ibm:cognos_controller:11.0.0:*:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "psirt@us.ibm.com",
      "ID": "CVE-2022-22364"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "IBM Cognos Controller 10.4.1, 10.4.2, and 11.0.0 is vulnerable to external service interaction attack, caused by improper validation of user-supplied input. A remote attacker could exploit this vulnerability to induce the application to perform server-side DNS lookups or HTTP requests to arbitrary domain names. By submitting suitable payloads, an attacker can cause the application server to attack other systems that it can interact with.  IBM X-Force ID:  220903."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-290"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/220903",
          "refsource": "",
          "tags": [
            "Vendor Advisory"
          ],
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/220903"
        },
        {
          "name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/220903",
          "refsource": "",
          "tags": [
            "Vendor Advisory"
          ],
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/220903"
        },
        {
          "name": "https://www.ibm.com/support/pages/node/7149876",
          "refsource": "",
          "tags": [
            "Vendor Advisory"
          ],
          "url": "https://www.ibm.com/support/pages/node/7149876"
        },
        {
          "name": "https://www.ibm.com/support/pages/node/7149876",
          "refsource": "",
          "tags": [
            "Vendor Advisory"
          ],
          "url": "https://www.ibm.com/support/pages/node/7149876"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV3": {
      "cvssV3": {
        "attackComplexity": "LOW",
        "attackVector": "NETWORK",
        "availabilityImpact": "NONE",
        "baseScore": 5.3,
        "baseSeverity": "MEDIUM",
        "confidentialityImpact": "NONE",
        "integrityImpact": "LOW",
        "privilegesRequired": "NONE",
        "scope": "UNCHANGED",
        "userInteraction": "NONE",
        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:L/A:N",
        "version": "3.1"
      },
      "exploitabilityScore": 3.9,
      "impactScore": 1.4
    }
  },
  "lastModifiedDate": "2025-01-07T20:16Z",
  "publishedDate": "2024-05-03T19:15Z"
}